### DISCUSSION TITLE
Grab metrics from a non prometheus endpoint

### DISCUSSION THREAD

**Discussion by duijva1** [signalfx/splunk-otel-collector#2327] [Q&A] [answered]
Hi all,

Is there a way to collect metrics from a non prometheus scraping port
We have a running container which published its data on TCP 9404

When we curl this port we do see metrics published.

This is the defined end point on the pod:

 - containerPort: 9404
      name: tcp-prometheus
      protocol: TCP

The metrics are ingested in to prometheus but we would like to see this in Splunk
Thanks

Arjan

**Comment by atoulme** [comment, marked as answer]
Sure, you can just provide the port and make sure to ingest those metrics using the prometheusreceiver.
